scripts/gulp/tasks/test-projects.ts

```diff
@@ -265,7 +265,9 @@ task('test:projects:typings', async () => {
 
   logger(`✔️ Temporary directory was created: ${tmpDirectory}`);
 
-  const dependencies = ['@types/react', '@types/react-dom', 'react', 'react-dom', 'typescript'].join(' ');
+  // Install dependencies, ensuring we specify the same TS version as our projects use
+  const tsVersion = fs.readJSONSync(paths.base('scripts', 'package.json')).devDependencies.typescript;
```

Comment:
can't you just

```tsx
const tsVersion = require('../package.json').devDependencies.typescript;
```

Reply:
With package.json that would work since it can't have comments, but I'm in the habit of using fs.readJSONSync since it handles commented JSON by default IIRC.